Source/ACE.Database/SerializedShardDatabase.cs

```diff
@@ -372,6 +372,25 @@ public List<Biota> GetHousesOwned()
             return _wrappedDatabase.GetAllegianceID(monarchID);
         }
 
+        public bool CanPurgeCharacter(uint characterId)
+        {
+            return _wrappedDatabase.CanPurgeCharacter(characterId);
+        }
+
+        public bool PurgeCharacter(uint characterId)
+        {
+            return _wrappedDatabase.PurgeCharacter(characterId);
+        }
+
+        public bool PurgeCharacters(int daysLimiter, out int numberOfCharactersPurged)
```

Comment:
this function shouldn't be in the serializedshard class, as it shouldn't be run at run time.